example.exs

```diff
@@ -38,6 +38,12 @@ sheet2 = %Sheet{name: 'Third', rows: [[1,2,3,4,5],
                                        ["hello", "world"]]}
          |> Sheet.set_row_height(3, 40)
 
+# If you need to merge cells, send a list of tuples,
+# where each tuple is a pair of start and end cell.
+# For example to merge cells from A1 to C1 use:
+# sheet = %Sheet{rows: rows, merge_cells: [{"A1", "C1"}]}
```

Comment:
Is it possible to integrate this into the actual executable code? (I typically run the libreoffice_test script in test/ to see if all the features appear to be working, I know this isn't optimal :/ but it would be great if that would show the expected result).
Does this work with vertical cells as well (or a combination of horizontal and vertical?) If so, would be great to include this in the example (one single "block" of merged cells should be OK), if not it would be great if you could add a quick note somewhere pointing out the limitations).